mxcube3/ui/actions/queue.js

```diff
@@ -238,73 +257,80 @@ export function sendStopQueue() {
 }
 
 
-export function sendMountSample(queueID) {
-  return function (dispatch) {
-    fetch(`mxcube/api/v0.1/sample_changer/${queueID}/mount`, {
-      method: 'PUT',
-      credentials: 'include',
-      headers: {
-        Accept: 'application/json',
-        'Content-type': 'application/json'
-      }
-    }).then((response) => {
-      if (response.status >= 400) {
-        throw new Error('Server refused to mount sample');
-      } else {
-        dispatch(mountSample(queueID));
-      }
-    });
-  };
+export function setQueueAction(queue) {
+  return { type: 'SET_QUEUE', queue };
```

Comment:
The action SET_QUEUE Its not used at the moment but as you note yourself there are situations when it perhaps should be used. Initialization could be one such case, that's why I added the action :). However I wanted to wait with that logic until we have decided how to, if at all exchange the state in local storage.